### PR TITLE
feature/css

### LINKED ISSUE #115: JavaFX: Priority FX
JavaFX for styling the priority.

### PR DESCRIPTION
**Pull request by BransonNg**

### CODE REVIEW

File: src/main/java/seedu/address/model/task/Done.java [hunk 1]
```diff
@@ -40,6 +40,10 @@ public static boolean isValidDone(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    public Boolean getIsDone() {
```
Comment (artlee06) [inline src/main/java/seedu/address/model/task/Done.java:43]: yes we need this for the tick

File: src/main/resources/view/LightTheme.css [hunk 2]
```diff
@@ -29,8 +29,7 @@
 }
 
 .label {
-	//-fx-font-family: "SofiaProRegular";
```
Comment (artlee06) [inline src/main/resources/view/LightTheme.css:32]: to be honest i have no idea why the // was there in the first place either

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -7,73 +7,76 @@
 <?import javafx.scene.control.MenuBar?>
 <?import javafx.scene.control.MenuItem?>
 <?import javafx.scene.control.SplitPane?>
+<?import javafx.scene.layout.GridPane?>
```
Comment (artlee06) [inline src/main/resources/view/MainWindow.fxml:10]: Whats the grid pane for by the way?

File: src/main/resources/view/PersonListCard.fxml [hunk 4]
```diff
@@ -9,32 +9,66 @@
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.RowConstraints?>
 <?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.CheckBox?>
+
+
+<HBox fx:id="cardPane" 
+    xmlns="http://javafx.com/javafx/8" 
+    xmlns:fx="http://javafx.com/fxml/1">
+    <GridPane HBox.hgrow="ALWAYS">
```
Comment (artlee06) [inline src/main/resources/view/PersonListCard.fxml:18]: What is the gridpane for exactly here? Is it just to add the extra margin/padding?

**Review by artlee06 — APPROVED** [review]
Just curious about the usage of gridpane but other than that great work I am excited!!

File: src/main/resources/view/MainWindow.fxml (hunk 3 above)
Comment (BransonNg) [reply src/main/resources/view/MainWindow.fxml:10]: Like display grid, let's us set grid columns of certain percentages

File: src/main/resources/view/PersonListCard.fxml (hunk 4 above)
Comment (BransonNg) [reply src/main/resources/view/PersonListCard.fxml:18]: Yes for the margin

**Comment by BransonNg** [comment]
image

**Pull request merged**